plugins/kernels/fps_kernels/routes.py

```diff
@@ -123,9 +127,9 @@ async def create_session(
     create_session = await request.json()
     kernel_name = create_session["kernel"]["name"]
     kernel_server = KernelServer(
-        kernelspec_path=str(
-            prefix_dir / "share" / "jupyter" / "kernels" / kernel_name / "kernel.json"
-        ),
+        kernelspec_path=(
+            prefix_dir / f"share/jupyter/kernels/{kernel_name}/kernel.json"
+        ).as_posix(),
```

Comment:
`kernelspec_path` is used to open a file in the back-end, is this going to work on Windows?

Reply:
Same as my [comment above](https://github.com/jupyter-server/jupyverse/pull/243#discussion_r1017558155), I think we should stick to `pathlib`'s `/` operator.